frontend/src/@types/UpdateUserProfile.ts

```diff
@@ -0,0 +1,5 @@
+export type UpdateUserProfile = {
+  name?: string;
+  preferredLanguageId?: number;
+  roleId?: number;
+};
```

Comment:
Can we refactor all these types and group them by bounded context? We should also remove the `src/interfaces.ts` file and place them under `@types`